webapp/src/app/pages/components/knowledge-graph/concept-map/concept-map.component.css

```
@@ -510,6 +509,77 @@ padding-right: 1rem;
+  color: #747d84;
+} 
+
+:host ::ng-deep .p-sidebar-header{
+  display: none;
+}
+
+:host ::ng-deep .p-sidebar-footer{
+  display: none;
+}
+
+:host ::ng-deep #recommendationButton {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)